src/components/CommentList.jsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getCommentsByArticle } from "../api";
+import AddComment from "./AddComment";
+
+
+const CommentList = () => {
+
+    const {article_id} = useParams()
+    const [commentList, setCommentList] = useState([])
+    const [isLoading,setIsLoading] = useState(true)
```

Comment:
Awesome to include the isLoading for the comments section, it's really important users know something is happening while on a slower connection.